### PR TITLE
Add basic-user to oc_setup

### PR DESCRIPTION
**Pull request by tonyskapunk**
##### SUMMARY

Include a 4th account using basic-user role.

##### ISSUE TYPE

- New Feature

##### Tests

- [x] TestBos2Sno: sno - https://www.distributed-ci.io/jobs/7f1aa833-6147-4b5e-99ef-35ea122761b9

---

Test-Hints: no-check

### CODE REVIEW

File: roles/oc_setup/README.md [hunk 1]
```diff
@@ -19,6 +18,17 @@ Access to a valid kubeconfig file via an `KUBECONFIG` environment variable.
 export KUBECONFIG=<kubeconfig_path>
 ```
 
+## Users
+
+These are the users created and the roles associated to them. See [official documentation about the default roles](https://docs.openshift.com/container-platform/4.17/post_installation_configuration/preparing-for-users.html#default-roles_post-install-preparing-for-users)
```
Comment (manurodriguez) [inline roles/oc_setup/README.md:23]: Now that docs.openshift.com would be decommisioned in Nov, maybe it's worth start using docs.redhat.com in our doc references, wdyt? the redirection would still work
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/postinstallation_configuration/post-install-preparing-for-users

**Review by manurodriguez — DISMISSED** [review]
LGTM

File: roles/oc_setup/README.md (hunk 1 above)
Comment (tonyskapunk) [reply roles/oc_setup/README.md:23]: Ahh, great point I'll make the change here

``` suggestion
## Users

These are the users created and the roles associated to them. See [official documentation about the default roles](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/postinstallation_configuration/post-install-preparing-for-users#default-roles_post-install-preparing-for-users)
```

**Review by manurodriguez — APPROVED** [review]
Thanks

**Review by betoredhat — APPROVED** [review]
lgtm!

**Pull request merged**